protobuffy.gemspec

```diff
@@ -26,7 +26,7 @@ require "protobuf/version"
   s.add_dependency 'thor'
   s.add_dependency 'thread_safe'
 
-  s.add_development_dependency 'ffi-rzmq'
+  s.add_development_dependency 'ffi-rzmq', '<= 2.0.4'
```

Comment:
For some reason 2.0.5 was breaking in Travis. 😞 

https://travis-ci.org/lookout/protobuffy/builds/227761107